### PR TITLE
ATC searches groups correctly

### PR DESCRIPTION
**Pull request by zealws**
```
# atc find 192.168.3.1
1: 192.168.3.0/24
```

### CODE REVIEW

**Comment by chantra** [comment]
thanks!
LGTM but I am going to ask if there is ways to cover this with unittests :)

**Comment by zfjagann** [comment]
Of course there are! I should have figured you'd ask for those. Should be able to do them later today.

**Comment by chantra** [comment]
:) thanks

File: src/daemon/atcd_test.go [hunk 1]
```diff
@@ -104,6 +104,19 @@ func TestAtcdGetGroupWith(_t *testing.T) {
...
+	grp := t.CreateGroup("1.2.3.0/24")
+
+	grp2 := t.GetGroupWith("1.2.3.4")
+
+	if grp2.ID != grp.ID {
+		t.Errorf("Group IDs don't match: %d != %d", grp.ID, grp2.ID)
+	}
+	t.checkMembers(grp2, "1.2.3.0/24")
+}
+
```
Comment (zfjagann) [inline src/daemon/atcd_test.go]: This was surprisingly easy.

**Comment by chantra** [comment]
\o/

**Pull request merged**